sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/LiveMetrics/DataCollection/DocumentHelper.cs

```diff
@@ -255,57 +261,105 @@ internal static ExceptionDocument ConvertToExceptionDocument(ActivityEvent activ
                 }
                 else if (tag.Key == SemanticConventions.AttributeExceptionType)
                 {
-                    exceptionType = tag.Value.ToString()!;
+                    exceptionDocument.ExceptionType = tag.Value.ToString()!;
                     continue;
                 }
                 else if (tag.Key == SemanticConventions.AttributeExceptionMessage)
                 {
-                    exceptionMessage = tag.Value.ToString()!;
+                    exceptionDocument.ExceptionMessage = tag.Value.ToString()!;
                     continue;
                 }
+                else if (tag.Key == SemanticConventions.AttributeExceptionStacktrace)
```

Comment:
If we do nothing shall we get rid of this check? 
This may not be from this PR, but I feel better to remove it now as tracking this small issues are tricky.

Reply:
I might have misunderstood too, are you planning to avoid huge string being added in properties? If so, update the comment.

Reply:
updated comment.